news/2023/2023-11-03-interview-with-ephemeral.md

```diff
@@ -0,0 +1,106 @@
+
+We've planned to add multiplayer medals for a long time, and I have so many ideas for those. I will share one briefly, it's more or less a medal that is contagious, you catch it from other people by playing in multiplayer rooms with people that have the medal. I'm really excited for that one!
+
+**Are there any rejected medal ideas that you're willing to share?**
+
+Skylord almost ended up there actually, which some people probably might suggest that it should still be there, you know, because it's really-really hard. 
+There also was a beatmap pack medal that was meant to be for Porter beatmap pack, but I don't recall if we signed Porter. I think we almost had Porter aboard as a Featured Artist, because he was in the same label that features Virtual Self. But I think it fell through and we haven't really pursued it since.
+
+**What's your favourite medal?**
+
+Probably Internment at this point. That was very-very fun to watch for me to be honest. It's cool when a medal has that much impact, because a lot of the time medals we released people just get in the first 18 hours, and then once the solution is on Osekai they forget about it.
+
+**Are there plans for challenge packs in other gamemodes?**
+
+Yep, absolutely. Once we get people willing to tutor us on the maps that would be good for that then yeah.
+
+**Any updates on the next batch of medals?**
+
+There may still be some coming out before the end of the year, maybe. They have not been made yet, but we would like to and it would be a big pack if it were to release and if it does not release this year it'll be early next year. And when I say big pack, I mean like all four modes. Really want to start getting all the modes in on it.
+
+**Any more HalfTime medals?**
+
+I figured people were tired of those after Aeon, so we haven't really thought about any since.
+
+**Any more plans for Loved map required medals?**
+
+I wasn't aware until like two packs ago that was even possible, I think someone in here told me that. I thought it was just literally Ranked maps only.
+
+**Do medals track the map you got them on?**
+
+They do, yeah, the whole score is saved, so it's there for us to use later on stuff.
+
+**What's your workout routine?**
+
+My workout routine is sit on a chair like a shrimp for like 18 hours a day.
+
+**Do you plan to make more meme-related medals?**
+
+If I can put a meme medal in that makes sense, I will at any given opportunity. It's just that you have to be careful when you put those in that you're not forcing the joke. I was worried a bit with 727 that it would be kind of like, you're beating a dead horse, but it's so timeless at this point.
+
+**Why can't you test medals?**
+
+I can't test this shit on my account, because the one time I did - you fuckers scraped it immediately, so I can't do that.
+
+**Which medal is worth the most internal score?**
+
+`Skylord`, `Beast Mode`, `Is This Real Life?` and `Final Boss` are all 1,000.
```

Comment:
Links to medal details on wiki or something?

Reply:
That would be funny to have, because Hush-Hush medals are kinda... secret you know.  I could link Osekai pages, but you'd murder me for that xD
Anyway's, I removed that question completely